experiments/training/README.md

```diff
@@ -0,0 +1,15 @@
+# Set up
```

Comment:
Can we make sure that all the set up for the project is withing the project? For e.g we are using part of the docker compose from kepler project and part of compose contents from peaks project. It is ok to use import and overide but I would recommend to have all the compose files within the peaks project.